tensorflow_addons/image/filters.py

```diff
@@ -204,3 +200,95 @@ def median_filter2d(
         output = tf.cast(median, image.dtype)
         output = img_utils.from_4D_image(output, original_ndims)
         return output
+
+
+@tf.function
+def gaussian_filter2d(
```

Comment:
Just for note we have already a gaussian kernel creator in Tensorflow but Is private https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/image_ops_impl.py#L3426